### PR TITLE
[FEAT] Flower UX

### PR DESCRIPTION
**Pull request by c-py**
# Description

Touch ups on Flower UX. 

Commented in line

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

localhost

# Checklist:

- [X] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [X] Screenshot if it includes any UI changes
- [X] I have read the contributing guidelines and agree to the T&Cs
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/island/buildings/components/building/market/Seeds.tsx [hunk 1]
```diff
@@ -234,6 +240,32 @@ export const Seeds: React.FC<Props> = ({ onClose }) => {
...
+                      onClick={() => onSeedClick(name)}
+                      image={ITEM_DETAILS[name].image}
+                      showOverlay={isSeedLocked(name)}
+                      secondaryImage={isSeedLocked(name) ? lock : undefined}
+                      count={inventory[name]}
+                    />
+                  ))}
+              </div>
+            </>
+          )}
```
Comment (c-py) [inline src/features/island/buildings/components/building/market/Seeds.tsx:268]: seed

File: src/features/island/flowers/FlowerBed.tsx [hunk 2]
```diff
@@ -64,11 +104,37 @@ export const FlowerBed: React.FC<Props> = ({ id }) => {
...
+    if (!hasHarvested) {
+      setShowCongratulationsModal(true);
+      return;
+    }
+
+    gameService.send({
+      type: "flower.harvested",
+      id,
+    });
+  };
```
Comment (c-py) [inline src/features/island/flowers/FlowerBed.tsx:133]: new_species

File: src/features/island/flowers/FlowerBedContent.tsx [hunk 3]
```diff
@@ -67,17 +70,48 @@ export const FlowerBedContent: React.FC<Props> = ({ id, onClose }) => {
...
+              src={
+                resultFlower
+                  ? ITEM_DETAILS[resultFlower].image
+                  : SUNNYSIDE.icons.search
+              }
+              className="h-4 mr-1"
+            />
+            <span className="text-xs">
+              {resultFlower ?? "Unknown combination"}
+            </span>
```
Comment (c-py) [inline src/features/island/flowers/FlowerBedContent.tsx:102]: ux1

File: src/features/island/hud/components/codex/pages/Flowers.tsx [hunk 4]
```diff
@@ -101,62 +99,19 @@ export const Flowers: React.FC<Props> = ({ onMilestoneReached }) => {
...
-                      className="h-full absolute inset-0 -top-1 mx-auto"
-                    />
-                  </div>
-                </div>
-              ))}
-            </div>
-          </>
-        )}
-      </Detail>
+      ></Detail>
```
Comment (c-py) [inline src/features/island/hud/components/codex/pages/Flowers.tsx:114]: codex

File: src/features/game/types/flowers.ts [hunk 5]
```diff
@@ -67,24 +75,83 @@ export const FLOWER_CROSS_BREED_AMOUNTS: Record<FlowerCrossBreedName, number> =
...
+type Lifecycle = {
+  seedling: any;
+  sprout: any;
+  halfway: any;
+  almost: any;
+  ready: any;
+};
+
+const IMAGES: Record<FlowerName, string> = {
+  "Red Pansy": "red_pansy",
```
Comment (brian-man) [inline src/features/game/types/flowers.ts:126]: TODO: images

File: src/features/island/flowers/FlowerBed.tsx [hunk 6]
```diff
@@ -3,33 +3,69 @@ import React, { useContext, useState } from "react";
...
+    id="congratulations"
+    className="absolute -top-1 left-1/2 -translate-x-1/2 h-36 w-40 pointer-events-none"
+  >
+    {Array.from({ length: 7 }).map((_, i) => (
+      <img
+        key={`flower-${i + 1}`}
+        src={ITEM_DETAILS[flowerName].image}
+        alt="Flower"
+        // Steal bee animations
+        className={`absolute left-1/2 -translate-x-1/2 swarm-bee-${i + 1}`}
```
Comment (spencerdezartsmith) [inline src/features/island/flowers/FlowerBed.tsx:44]: 👏🏼

**Review by spencerdezartsmith — APPROVED** [review]
Great job! Working well

**Pull request merged**